lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -121,7 +120,7 @@ def test_as_staff_for_honor(self):
         self.verify_response(params={'username': self.honor_user.username})
 
     def test_as_anonymous_user(self):
-        self.verify_response(expected_status_code=401)
+        self.verify_response(expected_status_code=200)
```

Comment:
sorry if i'm dense but I don't see any explicit change to permission logic in this PR; why has the handling for anonymous requests changed?